src/index.ts

```diff
@@ -2,7 +2,11 @@ import * as context from './context'
 export { context }
 
 export { setupWorker } from './setupWorker/setupWorker'
-export { SetupWorkerApi } from './setupWorker/glossary'
+export type {
+  SetupWorkerApi,
```

Comment:
What about the `SetupServerApi`? Isn't that needed as well?

Reply:
Good one! I didn't add it because it can be accessed via:

`import type { SetupServerApi } from 'msw/node`

WDYT?

Reply:
That's exactly how it should work. Nice!